SimCalorimetry/HcalSimAlgos/src/HcalSiPMHitResponse.cc

```diff
@@ -25,7 +25,7 @@ HcalSiPMHitResponse::HcalSiPMHitResponse(const CaloVSimParameterMap * parameterM
   //fill shape map
   shapeMap.emplace(HcalShapes::ZECOTEK,HcalShapes::ZECOTEK);
   shapeMap.emplace(HcalShapes::HAMAMATSU,HcalShapes::HAMAMATSU);
-  shapeMap.emplace(HcalShapes::HE2017,HcalShapes::HE2017);
```

Comment:
please keep the old shape in the map (since it might be requested in legacy simulation)